wiki/troubleshooting.md

```diff
@@ -8,6 +8,8 @@ To scan and repair those files, run `DISM /Online /Cleanup-Image /RestoreHealth`
 
 For more information about running DISM check [this page](https://support.microsoft.com/en-us/help/947821/fix-windows-update-errors-by-using-the-dism-or-system-update-readiness)
 
+Note: Running this command might take a while. If you just want to make sure nothing in the component store is not working, run `DISM /Online /Cleanup-Image /CheckHealth` instead before you continue troubleshooting.
```

Comment:
Use custom containers.
https://vitepress.dev/guide/markdown#custom-containers

Reply:
Done, thanks for the suggestion.